### DISCUSSION TITLE
Task 1 - LAB 05.01

### DISCUSSION THREAD

**Discussion by Ricardo-OB** [rramosp/ai4eng.v1#19] [05 - GESTIÓN DE MODELOS]
**EDIT: Tenía mal la formula para hallar el PNL con respecto a la predicción `p`, ya está todo bien**

Hola profe,
Tengo una duda con la Tarea 1 (LAB 05.01), ya tengo el arreglo con los PNL:

`s = [ 96.79, 101.52, 102.67, 103.91, 106.94, 100.11,  94.64,  89.25,
         95.76,  91.19,  93.69,  94.18,  87.18,  93.55, 101.44, 103.64,
        112.63, 104.79, 109.06, 111.38]`

`p = [1, 1, 0, 1, 1, 1, 1, 1, 1, 1, 1, 0, 1, 1, 1, 1, 0, 0, 1]`

![pnl](https://user-images.githubusercontent.com/64122293/120414715-04b0d680-c320-11eb-8eb7-9785503fc930.png)

Pero no entiendo muy bien los pasos a seguir después de hacer ese arreglo, muchas gracias, quedo atento.

**Comment by rramosp** [comment]
fíjate que este tipo de ejercicio es por valor de una variable (no tienes que completar una función como en la mayoría de los demás talleres)

el corrector verifica que el valor de la variable `pnl` sea el correcto respecto a la señal que se te generó a ti

es decir, entiendo que tal cual está deberías de poder ejecutar la celda para someter tu solución

**Reply by Ricardo-OB** [reply]
Hola profe, ya había arreglado el problema, no sé si leyó el EDIT. De todos modos gracias.

**Reply by rramosp** [reply]
listo. perfecto!!!
